disassemblers/ofrak_pyghidra/ofrak_pyghidra/standalone/pyghidra_analysis.py

```diff
@@ -337,7 +342,7 @@ def _decompile(func, flat_api):
     ifc.openProgram(flat_api.getCurrentProgram())
     res = ifc.decompileFunction(func, 0, TaskMonitor.DUMMY)
     if not res.decompileCompleted():
-        decomp = "Unable to decompile :("
+        raise Exception(f"Unable to decompile {func.getName()}")
```

Comment:
```suggestion
        raise RuntimeError(f"Unable to decompile {func.getName()}")
```

Reply:
upon review, i think there is an issue with raising an exception here at all. It is quite common for some functions to fail decompilation, and since the entire analysis happens in one step, it would fail the entire analysis of one decompilation fails.